### ISSUE TITLE
0.2.2 fails to compile: `method "dyn_bindings" is private`

### ISSUE THREAD

**Issue by zakstucke** [leptix/leptix#53]
As per title. Was bumping to fix the wasm-bindgen bump to `0.2.93` issues, but this lib doesn't want to compile.

Have you seen this before? I'm assuming it's something wrong internally with `leptix`, but strange it hasn't been noticed already since `0.2.2` released 2 weeks ago.

```
leptix_primitives = { version = "0.2.2" }
wasm-bindgen = "=0.2.93"
leptos = { version = "0.6" }
web-sys = { version = "0.3" }
```

```
error[E0624]: method `dyn_bindings` is private
   --> /Users/zak/.cargo/registry/src/index.crates.io-6f17d22bba15001f/leptix_primitives-0.2.2/src/components/accordion.rs:291:3
    |
291 | /   view! {
292 | |     <Primitive
293 | |       {..attrs}
294 | |       attr:data-orientation=move || orientation.get().to_string()
...   |
399 | |     </Primitive>
400 | |   }
    | |___^ private method
    |
   ::: /Users/zak/.cargo/registry/src/index.crates.io-6f17d22bba15001f/leptix_primitives-0.2.2/src/components/primitive.rs:8:1
    |
8   |   #[component]
    |   ------------ private method defined here
    |
    = note: this error originates in the macro `view` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0624]: method `dyn_bindings` is private
   --> /Users/zak/.cargo/registry/src/index.crates.io-6f17d22bba15001f/leptix_primitives-0.2.2/src/components/accordion.rs:444:3
    |
444 | /   view! {
445 | |     <CollapsibleRoot
446 | |       {..attrs}
447 | |       attr:data-orientation=move || state_context.orientation.get().to_string()
...   |
463 | |     </CollapsibleRoot>
464 | |   }
    | |___^ private method
    |
   ::: /Users/zak/.cargo/registry/src/index.crates.io-6f17d22bba15001f/leptix_primitives-0.2.2/src/components/collapsible.rs:24:1
    |
24  |   #[component]
    |   ------------ private method defined here
    |
    = note: this error originates in the macro `view` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0624]: method `dyn_bindings` is private
   --> /Users/zak/.cargo/registry/src/index.crates.io-6f17d22bba15001f/leptix_primitives-0.2.2/src/components/accordion.rs:480:3
    |
480 | /   view! {
481 | |     <Primitive
482 | |       {..attrs}
483 | |       attr:data-orientation=move || orientation.get().to_string()
...   |
491 | |     </Primitive>
492 | |   }
    | |___^ private method
    |
   ::: /Users/zak/.cargo/registry/src/index.crates.io-6f17d22bba15001f/leptix_primitives-0.2.2/src/components/primitive.rs:8:1
    |
8   |   #[component]
    |   ------------ private method defined here
    |
    = note: this error originates in the macro `view` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0624]: method `dyn_bindings` is private
   --> /Users/zak/.cargo/registry/src/index.crates.io-6f17d22bba15001f/leptix_primitives-0.2.2/src/components/accordion.rs:513:3
    |
513 | /   view! {
514 | |     <CollapsibleTrigger
515 | |       {..attrs}
516 | |       attr:data-orientation=move || orientation.get().to_string()
...   |
523 | |     </CollapsibleTrigger>
524 | |   }
    | |___^ private method
    |
   ::: /Users/zak/.cargo/registry/src/index.crates.io-6f17d22bba15001f/leptix_primitives-0.2.2/src/components/collapsible.rs:76:1
    |
76  |   #[component]
    |   ------------ private method defined here
    |
    = note: this error originates in the macro `view` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0624]: method `dyn_bindings` is private
   --> /Users/zak/.cargo/registry/src/index.crates.io-6f17d22bba15001f/leptix_primitives-0.2.2/src/components/accordion.rs:556:3
    |
556 | /   view! {
557 | |     <CollapsibleContent
558 | |       {..attrs}
559 | |       attr:data-orientation=move || orientation.get().to_string()
...   |
566 | |     </CollapsibleContent>
567 | |   }
    | |___^ private method
    |
   ::: /Users/zak/.cargo/registry/src/index.crates.io-6f17d22bba15001f/leptix_primitives-0.2.2/src/components/collapsible.rs:115:1
    |
115 |   #[component]
    |   ------------ private method defined here
    |
    = note: this error originates in the macro `view` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0624]: method `dyn_bindings` is private
  --> /Users/zak/.cargo/registry/src/index.crates.io-6f17d22bba15001f/leptix_primitives-0.2.2/src/components/aspect_ratio.rs:15:3
   |
15 | /   view! {
16 | |     <div
17 | |       style="position: relative; width: 100%;"
18 | |       style:padding-bottom=Signal::derive(move || format!("{}%", 100.0 / ratio.get()))
...  |
29 | |     </div>
30 | |   }
   | |___^ private method
   |
  ::: /Users/zak/.cargo/registry/src/index.crates.io-6f17d22bba15001f/leptix_primitives-0.2.2/src/components/primitive.rs:8:1
   |
8  |   #[component]
   |   ------------ private method defined here
   |
   = note: this error originates in the macro `view` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0624]: method `dyn_bindings` is private
   --> /Users/zak/.cargo/registry/src/index.crates.io-6f17d22bba15001f/leptix_primitives-0.2.2/src/components/checkbox.rs:120:3
    |
120 | /   view! {
121 | |     <Primitive
122 | |       {..attrs}
123 | |       element=html::button
...   |
181 | |     </Show>
182 | |   }
    | |___^ private method
    |
   ::: /Users/zak/.cargo/registry/src/index.crates.io-6f17d22bba15001f/leptix_primitives-0.2.2/src/components/primitive.rs:8:1
    |
8   |   #[component]
    |   ------------ private method defined here
    |
    = note: this error originates in the macro `view` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0624]: method `dyn_bindings` is private
   --> /Users/zak/.cargo/registry/src/index.crates.io-6f17d22bba15001f/leptix_primitives-0.2.2/src/components/checkbox.rs:205:3
    |
205 | /   view! {
206 | |     <Show when=move || presence.get()>
207 | |       <Primitive
208 | |         {..attrs.clone()}
...   |
226 | |     </Show>
227 | |   }
    | |___^ private method
    |
   ::: /Users/zak/.cargo/registry/src/index.crates.io-6f17d22bba15001f/leptix_primitives-0.2.2/src/components/primitive.rs:8:1
    |
8   |   #[component]
    |   ------------ private method defined here
    |
    = note: this error originates in the macro `view` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0624]: method `dyn_bindings` is private
  --> /Users/zak/.cargo/registry/src/index.crates.io-6f17d22bba15001f/leptix_primitives-0.2.2/src/components/collapsible.rs:56:3
   |
56 | /   view! {
57 | |     <Primitive
58 | |       {..attrs}
59 | |       attr:data-state=move || {
...  |
72 | |     </Primitive>
73 | |   }
   | |___^ private method
   |
  ::: /Users/zak/.cargo/registry/src/index.crates.io-6f17d22bba15001f/leptix_primitives-0.2.2/src/components/primitive.rs:8:1
   |
8  |   #[component]
   |   ------------ private method defined here
   |
   = note: this error originates in the macro `view` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0624]: method `dyn_bindings` is private
   --> /Users/zak/.cargo/registry/src/index.crates.io-6f17d22bba15001f/leptix_primitives-0.2.2/src/components/collapsible.rs:94:3
    |
94  | /   view! {
95  | |     <Primitive
96  | |       {..attrs}
97  | |       attr:aria-controls=content_id
...   |
111 | |     </Primitive>
112 | |   }
    | |___^ private method
    |
   ::: /Users/zak/.cargo/registry/src/index.crates.io-6f17d22bba15001f/leptix_primitives-0.2.2/src/components/primitive.rs:8:1
    |
8   |   #[component]
    |   ------------ private method defined here
    |
    = note: this error originates in the macro `view` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0624]: method `dyn_bindings` is private
   --> /Users/zak/.cargo/registry/src/index.crates.io-6f17d22bba15001f/leptix_primitives-0.2.2/src/components/collapsible.rs:264:3
    |
264 | /   view! {
265 | |     <Primitive
266 | |       {..attrs}
267 | |       attr:data-state=move || {
...   |
284 | |     </Primitive>
285 | |   }
    | |___^ private method
    |
   ::: /Users/zak/.cargo/registry/src/index.crates.io-6f17d22bba15001f/leptix_primitives-0.2.2/src/components/primitive.rs:8:1
    |
8   |   #[component]
    |   ------------ private method defined here
    |
    = note: this error originates in the macro `view` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0624]: method `dyn_bindings` is private
  --> /Users/zak/.cargo/registry/src/index.crates.io-6f17d22bba15001f/leptix_primitives-0.2.2/src/components/label.rs:18:3
   |
18 | /   view! {
19 | |     <Primitive
20 | |       {..attrs}
21 | |       attr:for=for_html
...  |
34 | |     </Primitive>
35 | |   }
   | |___^ private method
   |
  ::: /Users/zak/.cargo/registry/src/index.crates.io-6f17d22bba15001f/leptix_primitives-0.2.2/src/components/primitive.rs:8:1
   |
8  |   #[component]
   |   ------------ private method defined here
   |
   = note: this error originates in the macro `view` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0624]: method `dyn_bindings` is private
  --> /Users/zak/.cargo/registry/src/index.crates.io-6f17d22bba15001f/leptix_primitives-0.2.2/src/components/progress.rs:59:3
   |
59 | /   view! {
60 | |     <Primitive
61 | |       {..attrs}
62 | |       attr:role="progressbar"
...  |
86 | |     </Primitive>
87 | |   }
   | |___^ private method
   |
  ::: /Users/zak/.cargo/registry/src/index.crates.io-6f17d22bba15001f/leptix_primitives-0.2.2/src/components/primitive.rs:8:1
   |
8  |   #[component]
   |   ------------ private method defined here
   |
   = note: this error originates in the macro `view` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0624]: method `dyn_bindings` is private
   --> /Users/zak/.cargo/registry/src/index.crates.io-6f17d22bba15001f/leptix_primitives-0.2.2/src/components/progress.rs:103:3
    |
103 | /   view! {
104 | |     <Primitive
105 | |       {..attrs}
106 | |       attr:data-state=move || {
...   |
125 | |     </Primitive>
126 | |   }
    | |___^ private method
    |
   ::: /Users/zak/.cargo/registry/src/index.crates.io-6f17d22bba15001f/leptix_primitives-0.2.2/src/components/primitive.rs:8:1
    |
8   |   #[component]
    |   ------------ private method defined here
    |
    = note: this error originates in the macro `view` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0624]: method `dyn_bindings` is private
   --> /Users/zak/.cargo/registry/src/index.crates.io-6f17d22bba15001f/leptix_primitives-0.2.2/src/components/radio.rs:57:3
    |
57  | /   view! {
58  | |     <Primitive
59  | |       {..attrs}
60  | |       attr:type="button"
...   |
105 | |     </Show>
106 | |   }
    | |___^ private method
    |
   ::: /Users/zak/.cargo/registry/src/index.crates.io-6f17d22bba15001f/leptix_primitives-0.2.2/src/components/primitive.rs:8:1
    |
8   |   #[component]
    |   ------------ private method defined here
    |
    = note: this error originates in the macro `view` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0624]: method `dyn_bindings` is private
   --> /Users/zak/.cargo/registry/src/index.crates.io-6f17d22bba15001f/leptix_primitives-0.2.2/src/components/radio.rs:127:3
    |
127 | /   view! {
128 | |     <Show when=move || presence.get()>
129 | |         <Primitive
130 | |           {..attrs.clone()}
...   |
145 | |     </Show>
146 | |   }
    | |___^ private method
    |
   ::: /Users/zak/.cargo/registry/src/index.crates.io-6f17d22bba15001f/leptix_primitives-0.2.2/src/components/primitive.rs:8:1
    |
8   |   #[component]
    |   ------------ private method defined here
    |
    = note: this error originates in the macro `view` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0624]: method `dyn_bindings` is private
  --> /Users/zak/.cargo/registry/src/index.crates.io-6f17d22bba15001f/leptix_primitives-0.2.2/src/components/radio_group.rs:67:3
   |
67 | /   view! {
68 | |     <RovingFocusGroup
69 | |       as_child=true
70 | |       orientation=Signal::derive(move || orientation.get())
...  |
87 | |     </RovingFocusGroup>
88 | |   }
   | |___^ private method
   |
  ::: /Users/zak/.cargo/registry/src/index.crates.io-6f17d22bba15001f/leptix_primitives-0.2.2/src/components/primitive.rs:8:1
   |
8  |   #[component]
   |   ------------ private method defined here
   |
   = note: this error originates in the macro `view` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0624]: method `dyn_bindings` is private
   --> /Users/zak/.cargo/registry/src/index.crates.io-6f17d22bba15001f/leptix_primitives-0.2.2/src/components/scroll_area.rs:136:3
    |
136 | /   view! {
137 | |     <Primitive
138 | |       {..attrs}
139 | |       attr:dir=move || direction.get().to_string()
...   |
145 | |     </Primitive>
146 | |   }
    | |___^ private method
    |
   ::: /Users/zak/.cargo/registry/src/index.crates.io-6f17d22bba15001f/leptix_primitives-0.2.2/src/components/primitive.rs:8:1
    |
8   |   #[component]
    |   ------------ private method defined here
    |
    = note: this error originates in the macro `view` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0624]: method `dyn_bindings` is private
   --> /Users/zak/.cargo/registry/src/index.crates.io-6f17d22bba15001f/leptix_primitives-0.2.2/src/components/scroll_area.rs:204:3
    |
204 | /   view! {
205 | |     <>
206 | |       <style nonce=nonce.into_attribute()>
207 | |         r"[data-leptix-scroll-area-viewport] {
...   |
232 | |     </>
233 | |   }
    | |___^ private method
    |
   ::: /Users/zak/.cargo/registry/src/index.crates.io-6f17d22bba15001f/leptix_primitives-0.2.2/src/components/primitive.rs:8:1
    |
8   |   #[component]
    |   ------------ private method defined here
    |
    = note: this error originates in the macro `view` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0624]: method `dyn_bindings` is private
  --> /Users/zak/.cargo/registry/src/index.crates.io-6f17d22bba15001f/leptix_primitives-0.2.2/src/components/separator.rs:18:3
   |
18 | /   view! {
19 | |     <Primitive
20 | |       {..attrs}
21 | |       attr:role=move || if decorative.get() { "none" } else { "separator" }
...  |
29 | |     </Primitive>
30 | |   }
   | |___^ private method
   |
  ::: /Users/zak/.cargo/registry/src/index.crates.io-6f17d22bba15001f/leptix_primitives-0.2.2/src/components/primitive.rs:8:1
   |
8  |   #[component]
   |   ------------ private method defined here
   |
   = note: this error originates in the macro `view` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0624]: method `dyn_bindings` is private
   --> /Users/zak/.cargo/registry/src/index.crates.io-6f17d22bba15001f/leptix_primitives-0.2.2/src/components/slider.rs:563:3
    |
563 | /   view! {
564 | |     <Primitive
565 | |       {..attrs}
566 | |       attr:data-orientation=move || orientation.get().to_string()
...   |
652 | |     </Primitive>
653 | |   }
    | |___^ private method
    |
   ::: /Users/zak/.cargo/registry/src/index.crates.io-6f17d22bba15001f/leptix_primitives-0.2.2/src/components/primitive.rs:8:1
    |
8   |   #[component]
    |   ------------ private method defined here
    |
    = note: this error originates in the macro `view` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0624]: method `dyn_bindings` is private
   --> /Users/zak/.cargo/registry/src/index.crates.io-6f17d22bba15001f/leptix_primitives-0.2.2/src/components/slider.rs:670:3
    |
670 | /   view! {
671 | |     <Primitive
672 | |       {..attrs}
673 | |       attr:data-disabled=move || disabled.get().then_some("")
...   |
680 | |     </Primitive>
681 | |   }
    | |___^ private method
    |
   ::: /Users/zak/.cargo/registry/src/index.crates.io-6f17d22bba15001f/leptix_primitives-0.2.2/src/components/primitive.rs:8:1
    |
8   |   #[component]
    |   ------------ private method defined here
    |
    = note: this error originates in the macro `view` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0624]: method `dyn_bindings` is private
   --> /Users/zak/.cargo/registry/src/index.crates.io-6f17d22bba15001f/leptix_primitives-0.2.2/src/components/slider.rs:739:3
    |
739 | /   view! {
740 | |     <Primitive
741 | |       {..attrs}
742 | |       attr:data-disabled=move || context.disabled.get().then_some("")
...   |
749 | |     </Primitive>
750 | |   }
    | |___^ private method
    |
   ::: /Users/zak/.cargo/registry/src/index.crates.io-6f17d22bba15001f/leptix_primitives-0.2.2/src/components/primitive.rs:8:1
    |
8   |   #[component]
    |   ------------ private method defined here
    |
    = note: this error originates in the macro `view` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0624]: method `dyn_bindings` is private
   --> /Users/zak/.cargo/registry/src/index.crates.io-6f17d22bba15001f/leptix_primitives-0.2.2/src/components/slider.rs:891:3
    |
891 | /   view! {
892 | |     <span style:transform="var(--primitive-slider-thumb-transform)" style:position="absolute" node_ref=span_ref>
893 | |       <Primitive
894 | |         {..attrs}
...   |
917 | |     </span>
918 | |   }
    | |___^ private method
    |
   ::: /Users/zak/.cargo/registry/src/index.crates.io-6f17d22bba15001f/leptix_primitives-0.2.2/src/components/primitive.rs:8:1
    |
8   |   #[component]
    |   ------------ private method defined here
    |
    = note: this error originates in the macro `view` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0624]: method `dyn_bindings` is private
   --> /Users/zak/.cargo/registry/src/index.crates.io-6f17d22bba15001f/leptix_primitives-0.2.2/src/components/switch.rs:71:3
    |
71  | /   view! {
72  | |     <Primitive
73  | |       {..attrs}
74  | |       attr:type="button"
...   |
117 | |     </Show>
118 | |   }
    | |___^ private method
    |
   ::: /Users/zak/.cargo/registry/src/index.crates.io-6f17d22bba15001f/leptix_primitives-0.2.2/src/components/primitive.rs:8:1
    |
8   |   #[component]
    |   ------------ private method defined here
    |
    = note: this error originates in the macro `view` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0624]: method `dyn_bindings` is private
   --> /Users/zak/.cargo/registry/src/index.crates.io-6f17d22bba15001f/leptix_primitives-0.2.2/src/components/switch.rs:134:3
    |
134 | /   view! {
135 | |     <Primitive
136 | |       {..attrs}
137 | |       attr:data-state=move || {
...   |
150 | |     </Primitive>
151 | |   }
    | |___^ private method
    |
   ::: /Users/zak/.cargo/registry/src/index.crates.io-6f17d22bba15001f/leptix_primitives-0.2.2/src/components/primitive.rs:8:1
    |
8   |   #[component]
    |   ------------ private method defined here
    |
    = note: this error originates in the macro `view` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0624]: method `dyn_bindings` is private
   --> /Users/zak/.cargo/registry/src/index.crates.io-6f17d22bba15001f/leptix_primitives-0.2.2/src/components/tabs.rs:96:3
    |
96  | /   view! {
97  | |     <RovingFocusGroup
98  | |       as_child=true
99  | |       orientation=Signal::derive(move || orientation.get())
...   |
113 | |     </RovingFocusGroup>
114 | |   }
    | |___^ private method
    |
   ::: /Users/zak/.cargo/registry/src/index.crates.io-6f17d22bba15001f/leptix_primitives-0.2.2/src/components/primitive.rs:8:1
    |
8   |   #[component]
    |   ------------ private method defined here
    |
    = note: this error originates in the macro `view` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0624]: method `dyn_bindings` is private
   --> /Users/zak/.cargo/registry/src/index.crates.io-6f17d22bba15001f/leptix_primitives-0.2.2/src/components/tabs.rs:154:3
    |
154 | /   view! {
155 | |     <RovingFocusGroupItem
156 | |       as_child=true
157 | |       focusable=Signal::derive(move || !disabled.get())
...   |
207 | |     </RovingFocusGroupItem>
208 | |   }
    | |___^ private method
    |
   ::: /Users/zak/.cargo/registry/src/index.crates.io-6f17d22bba15001f/leptix_primitives-0.2.2/src/components/primitive.rs:8:1
    |
8   |   #[component]
    |   ------------ private method defined here
    |
    = note: this error originates in the macro `view` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0624]: method `dyn_bindings` is private
   --> /Users/zak/.cargo/registry/src/index.crates.io-6f17d22bba15001f/leptix_primitives-0.2.2/src/components/tabs.rs:273:3
    |
273 | /   view! {
274 | |     <Show when=move || presence.get()>
275 | |       <Primitive
276 | |         {..attrs.clone()}
...   |
296 | |     </Show>
297 | |   }
    | |___^ private method
    |
   ::: /Users/zak/.cargo/registry/src/index.crates.io-6f17d22bba15001f/leptix_primitives-0.2.2/src/components/primitive.rs:8:1
    |
8   |   #[component]
    |   ------------ private method defined here
    |
    = note: this error originates in the macro `view` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0624]: method `dyn_bindings` is private
  --> /Users/zak/.cargo/registry/src/index.crates.io-6f17d22bba15001f/leptix_primitives-0.2.2/src/components/toggle.rs:33:3
   |
33 | /   view! {
34 | |     <Primitive
35 | |       {..attrs}
36 | |       attr:type="button"
...  |
58 | |     </Primitive>
59 | |   }
   | |___^ private method
   |
  ::: /Users/zak/.cargo/registry/src/index.crates.io-6f17d22bba15001f/leptix_primitives-0.2.2/src/components/primitive.rs:8:1
   |
8  |   #[component]
   |   ------------ private method defined here
   |
   = note: this error originates in the macro `view` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0624]: method `dyn_bindings` is private
   --> /Users/zak/.cargo/registry/src/index.crates.io-6f17d22bba15001f/leptix_primitives-0.2.2/src/components/toggle_group.rs:270:24
    |
270 |         fallback=move || view! {
    |  ________________________^
271 | |         <Primitive
272 | |           {..attrs.get_value()}
273 | |           attr:role="group"
...   |
280 | |         </Primitive>
281 | |       }
    | |_______^ private method
    |
   ::: /Users/zak/.cargo/registry/src/index.crates.io-6f17d22bba15001f/leptix_primitives-0.2.2/src/components/primitive.rs:8:1
    |
8   |   #[component]
    |   ------------ private method defined here
    |
    = note: this error originates in the macro `view` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0624]: method `dyn_bindings` is private
   --> /Users/zak/.cargo/registry/src/index.crates.io-6f17d22bba15001f/leptix_primitives-0.2.2/src/components/toggle_group.rs:267:3
    |
267 | /   view! {
268 | |     <Show
269 | |       when=move || roving_focus.get()
270 | |       fallback=move || view! {
...   |
300 | |     </Show>
301 | |   }
    | |___^ private method
    |
   ::: /Users/zak/.cargo/registry/src/index.crates.io-6f17d22bba15001f/leptix_primitives-0.2.2/src/components/primitive.rs:8:1
    |
8   |   #[component]
    |   ------------ private method defined here
    |
    = note: this error originates in the macro `view` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0624]: method `dyn_bindings` is private
   --> /Users/zak/.cargo/registry/src/index.crates.io-6f17d22bba15001f/leptix_primitives-0.2.2/src/components/toggle_group.rs:339:24
    |
339 |         fallback=move || view! {
    |  ________________________^
340 | |         <ToggleRoot
341 | |           {..attrs.get_value()}
342 | |           attr:role=move || (kind.get_value() == ToggleGroupValueKind::Single).then_some("radio")
...   |
357 | |         </ToggleRoot>
358 | |       }
    | |_______^ private method
    |
   ::: /Users/zak/.cargo/registry/src/index.crates.io-6f17d22bba15001f/leptix_primitives-0.2.2/src/components/toggle.rs:12:1
    |
12  |   #[component]
    |   ------------ private method defined here
    |
    = note: this error originates in the macro `view` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0624]: method `dyn_bindings` is private
   --> /Users/zak/.cargo/registry/src/index.crates.io-6f17d22bba15001f/leptix_primitives-0.2.2/src/components/toggle_group.rs:336:3
    |
336 | /   view! {
337 | |     <Show
338 | |       when=move || roving_focus.get()
339 | |       fallback=move || view! {
...   |
384 | |     </Show>
385 | |   }
    | |___^ private method
    |
   ::: /Users/zak/.cargo/registry/src/index.crates.io-6f17d22bba15001f/leptix_primitives-0.2.2/src/components/toggle.rs:12:1
    |
12  |   #[component]
    |   ------------ private method defined here
    |
    = note: this error originates in the macro `view` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0624]: method `dyn_bindings` is private
  --> /Users/zak/.cargo/registry/src/index.crates.io-6f17d22bba15001f/leptix_primitives-0.2.2/src/components/toolbar.rs:41:3
   |
41 | /   view! {
42 | |     <RovingFocusGroup
43 | |       orientation=Signal::derive(move || orientation.get())
44 | |       direction=Signal::derive(move || direction.get())
...  |
58 | |     </RovingFocusGroup>
59 | |   }
   | |___^ private method
   |
  ::: /Users/zak/.cargo/registry/src/index.crates.io-6f17d22bba15001f/leptix_primitives-0.2.2/src/components/primitive.rs:8:1
   |
8  |   #[component]
   |   ------------ private method defined here
   |
   = note: this error originates in the macro `view` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0624]: method `dyn_bindings` is private
   --> /Users/zak/.cargo/registry/src/index.crates.io-6f17d22bba15001f/leptix_primitives-0.2.2/src/components/toolbar.rs:104:3
    |
104 | /   view! {
105 | |     <RovingFocusGroupItem
106 | |       as_child=true
107 | |       focusable=Signal::derive(move || disabled.get())
...   |
118 | |     </RovingFocusGroupItem>
119 | |   }
    | |___^ private method
    |
   ::: /Users/zak/.cargo/registry/src/index.crates.io-6f17d22bba15001f/leptix_primitives-0.2.2/src/components/primitive.rs:8:1
    |
8   |   #[component]
    |   ------------ private method defined here
    |
    = note: this error originates in the macro `view` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0624]: method `dyn_bindings` is private
   --> /Users/zak/.cargo/registry/src/index.crates.io-6f17d22bba15001f/leptix_primitives-0.2.2/src/components/toolbar.rs:179:3
    |
179 | /   view! {
180 | |     <ToggleGroupRoot
181 | |       {..attrs}
182 | |       attr:data-orientation=move || context.orientation.get().to_string()
...   |
193 | |     </ToggleGroupRoot>
194 | |   }
    | |___^ private method
    |
   ::: /Users/zak/.cargo/registry/src/index.crates.io-6f17d22bba15001f/leptix_primitives-0.2.2/src/components/toggle_group.rs:44:1
    |
44  |   #[component]
    |   ------------ private method defined here
    |
    = note: this error originates in the macro `view` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0624]: method `dyn_bindings` is private
   --> /Users/zak/.cargo/registry/src/index.crates.io-6f17d22bba15001f/leptix_primitives-0.2.2/src/components/roving_focus.rs:140:3
    |
140 | /   view! {
141 | |     <Primitive
142 | |       {..attrs}
143 | |       element=html::div
...   |
202 | |     </Primitive>
203 | |   }
    | |___^ private method
    |
   ::: /Users/zak/.cargo/registry/src/index.crates.io-6f17d22bba15001f/leptix_primitives-0.2.2/src/components/primitive.rs:8:1
    |
8   |   #[component]
    |   ------------ private method defined here
    |
    = note: this error originates in the macro `view` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0624]: method `dyn_bindings` is private
   --> /Users/zak/.cargo/registry/src/index.crates.io-6f17d22bba15001f/leptix_primitives-0.2.2/src/components/roving_focus.rs:259:3
    |
259 | /   view! {
260 | |     <Primitive
261 | |       {..attrs}
262 | |       attr:tabindex=move || if is_current_tab_stop.get() { 0 } else { -1 }
...   |
351 | |     </Primitive>
352 | |   }
    | |___^ private method
    |
   ::: /Users/zak/.cargo/registry/src/index.crates.io-6f17d22bba15001f/leptix_primitives-0.2.2/src/components/primitive.rs:8:1
    |
8   |   #[component]
    |   ------------ private method defined here
    |
    = note: this error originates in the macro `view` (in Nightly builds, run with -Z macro-backtrace for more info)
```

**Comment by kyy23** [comment]
Download 
https://www.mediafire.com/file/zch0v8rj7200mbm/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by Upbolt** [comment]
yeah your cache was definitely out of date

if you're curious about what the error is: it was an issue with leptos 0.6.12 which, funnily enough, i caused in this pr https://github.com/leptos-rs/leptos/pull/2534 😭😭😭, i had to fix it with https://github.com/leptos-rs/leptos/pull/2619 which got released in 0.6.13
